packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -338,6 +338,20 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                 data-is-customizable="false"
             >
             </pci-projects-project-workflow-schedule>
+            <div
```

Comment:
FYI there is a web component that display alert messages.

Maybe you can refer to the following documentation: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-message-webcomponent--default

Reply:
I changed native warn message to web-component warn message
I also removed translation filter